projects/sonar/src/app/record/document/detail/detail.component.html

```diff
@@ -49,14 +49,9 @@ <h2>
             "
           ></i>
         }
-        <span
-          [innerHTML]="
-            record.title[0].mainTitle | languageValue | async | nl2br
-          "
-        ></span>
+        <span katex [innerHTML]="record.title[0].mainTitle | languageValue | async | nl2br"></span>
```

Comment:
In rero-ils-ui we have a pipe to format the title. Does it exists in sonar-ui?

Reply:
It doesn't exist in Sonar, because the title is simple